core/server/data/importer/importers/data/settings.js

```diff
@@ -204,6 +207,21 @@ class SettingsImporter extends BaseImporter {
         }
 
         _.each(this.dataToImport, (obj) => {
+            if (obj.key === 'labs' && obj.value) {
+                const inputLabsValue = JSON.parse(obj.value);
+                const filteredLabsValue = {};
+
+                for (const flag in inputLabsValue) {
+                    if (WRITABLE_KEYS_ALLOWLIST.includes(flag)) {
+                        filteredLabsValue[flag] = inputLabsValue[flag];
+                    }
+                }
+
+                // Overwrite the labs setting with our current defaults
+                // Ensures things that are enabled in new versions, are turned on
+                obj.value = JSON.stringify(_.assign({}, filteredLabsValue, labsDefaults));
```

Comment:
This would have the effect of re-enabling labs features that have been explicitly disabled, is that intentional?

Reply:
Yup. This is how the rules have been set up in the past as well - https://github.com/TryGhost/Ghost/commit/ea6d656457eb9088c5a121fc8b3a4920a4e1b903#diff-175e0a6e8ad6003197e8359cb8a50b37dfa14cd6eab87b67840c63cd1399386aL189. 

Maybe we'd want to change it, but for now I'm trying to keep the restoration to the minimum with minor improvements to avoid common pitfalls we had in the past.